### ISSUE TITLE
NVDA Sometimes Crashes on First Launch After Reboot

### ISSUE THREAD

**Issue by rob-aph** [nvaccess/nvda#13700]
### Steps to reproduce:
1. I am experiencing this problem on two different machines both running the same versions of Windows.
2. Reboot PC and use JAWS for a while. Check email, web browse, edit text files, etc.
3. Quit JAWS and attempt to start NVDA.
4. NVDA startup sound is played, time passes, the first syllable of the phrase "Loading NVDA" is spoken, then NVDA crashes.
### Actual behavior:
NVDA fails to run.
### Expected behavior:
NVDA should run normally.
#### NVDA installed/portable/running from source:
NVDA installed.
#### NVDA version:
2022.1rc1 - crash also happend with beta's leading up to the rc release.
#### Windows version:
Version 21H2 (OS Build 19044.1706)
#### Name and version of other software in use when reproducing the issue:
Outlook 16.0.15128.20178 
Notepad++ 8.4, 32-bit
Windows Command Prompt
Microsoft Edge (always latest)
#### Other information about your system:
Both machines are Dell, with different motherboards -- one is a business unit, other is a home unit.
#### Does the issue still occur after restarting your computer?
It only occurs after a restart, and doesn't always occur. Running NVDA for a second time is always fine.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The last two betas leading up to this rc had the same problem.
#### If NVDA add-ons are disabled, is your problem still occurring?
I don't have any add-ons installed.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.
[nvda.log](https://github.com/nvaccess/nvda/files/8699552/nvda.log)

**Comment by josephsl** [comment]
Hi, duplicate of #13694, a fix is being developed. Thanks.

**Comment by rob-aph** [comment]
Closed as is duplicate of #13694,
